packages/addons/service/tvheadend42/source/defaults/config

```diff
@@ -1,13 +1,14 @@
 {
-	"uilevel": 1,
-	"uilevel_nochange": false,
+	"wizard": "hello",
+	"uilevel": 0,
```

Comment:
The main problem is that the users don't understand the view levels (or didn't know that they exist) so I'd like to keep the advanced level that at least basic configuration is possible (also a lot/all tuts are on base of expert or 4.0 - so if you have no idea that view levels are existing and 4/5 of the configs from the tutorial are not there isn't that helpful). 
The `uilevel_nochange": true` would also prohibit that users can change that easily. I don't know the real perfect setting here, but from feedback the current setting is "okayish" (ofc it didn't fit to everyone).
pls change back to `uilevel": 1`